frontend/src/shared/layout/navigation/side-navigation.tsx

```diff
@@ -152,7 +152,7 @@ export default function SideNavigation () {
                         selectedOption={selectedOption}
                         onChange={({ detail }) => {
                             setSelectedOption(detail.selectedOption);
-                            dispatch(setItemsForProjectName(detail.selectedOption.value));
+                            dispatch(setItemsForProjectName({ 'project': detail.selectedOption.value, 'enabledServices': applicationConfig.configuration.EnabledServices }));
```

Comment:
Same comment as `project-detail`

Reply:
Especially now that this is a typed object maybe we shouldn't wrap the object properties in strings for all these calls? Feels unnecessary. Probably not worth changing unless you're making another PR though.